MEAnalysis/python/MEAnalysis_heppy.py

```diff
@@ -23,6 +23,9 @@
     print "Loading ME config from", os.environ["ME_CONF"]
     meconf = imp.load_source("meconf", os.environ["ME_CONF"])
     from meconf import Conf
+elif 0:
```

Comment:
@d-salerno why is this `elif 0` here? if you want to run tests locally, you could write it in the Makefile, like
~~~
test_MEAnalysis_FH:
        ME_CONF=MEAnalysis/python/cfg_FH.py python MEAnalysis/python/MEAnalysis_heppy.py
~~~

let's avoid strange looking dead code that we always have to remember to comment in or out.